src/main/java/seedu/address/model/event/Event.java

```diff
@@ -48,6 +48,30 @@ public Event(EventName name, EventDate date, EventTime time, boolean isDone) {
         this.isDone = isDone;
     }
 
+
+    /**
+     * This is an overloaded constructor of an Event.
+     *
+     * @param name of the Event.
+     * @param date at which the Event occurs.
+     * @param time of the Event.
+     * @param isDone A boolean to indicate is the event is done.
+     * @param participants participants to be added if not found
+     */
+    public Event(EventName name, EventDate date, EventTime time, boolean isDone, List<Participant> participants) {
+        this.eventName = name;
+        this.eventDate = date;
+        this.eventTime = time;
+        this.isDone = isDone;
+        if (participants != null) {
```

Comment:
`participants` should require non null I think. An empty list can be passed as an argument instead if there are no participants added for this event yet.
It would also make this part of the code cleaner.